src/Form/Company/RegistrationType.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Form\Company;
+
+use AppBundle\Form\AddressType;
+use AppBundle\Message\Company\RequestRegistration;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\DataMapperInterface;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RegistrationType extends AbstractType implements DataMapperInterface
```

Comment:
Just curious: why do we need to implement `DataMapperInterface`?
Usually I do this:

```php
public function configureOptions(OptionsResolver $resolver)
{
    $resolver->setDefaults(array(
        'data_class' => RequestRegistration::class,
    ));
}
```

Reply:
"RequestRegistration" has elements in the constructor. If I pass it as you put it, I would have to remove the arguments and use setter, which makes the class anemic.

Reply:
```DataMapperInterface``` allows you to implement 2 methods to match the need that the class has to function properly.